play/settings/settings.py

```diff
@@ -79,14 +88,21 @@
 # Database
 # https://docs.djangoproject.com/en/2.0/ref/settings/#databases
 
+DB_NAME = get_env("POSTGRES_DB", "battlesnakeio_play", True)
+DB_USER = get_env("POSTGRES_USER", None, False)
+DB_PASS = get_env("POSTGRES_PASSWORD", None, False)
+DB_HOST = get_env("BATTLESNAKEIO_POSTGRES_HOST", None, False)
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        'ENGINE': 'django.db.backends.postgresql',
```

Comment:
what do you think about running just sqlite locally, and using postgres in the cluster?

Reply:
seems sensible. I will make it mounted in a volume so we can persist data locally.